DistantWorlds2.ModLoader.Core/IModManager.cs

```diff
@@ -20,6 +20,8 @@ public interface IModManager
 
     ConcurrentDictionary<string, object> SharedVariables { get; }
 
+    public string Checksum { get; }
```

Comment:
```suggestion
    // Game data checksum
    public string Checksum { get; }
```
or...
```suggestion
    public string GameDataChecksum { get; }
```
(would require refactoring)

N.B.